src/resolvers/hosts/hosts.integration.ts

```diff
@@ -387,6 +388,39 @@ describe('hosts query', function () {
             });
         });
 
+        describe('modified_on', function () {
+            test('lte', async () => {
+                const { data } = await runQuery(BASIC_QUERY,
+                    { filter: { modified_on: {
+                        lte: '2019-01-10T08:07:03.354312Z'
+                    }}}
+                );
+                data.hosts.data.should.have.length(1);
+                data.hosts.data[0].id.should.equal(TEST_ORG_ID_HOST_IDS[0]);
+            });
+
+            test('gte', async () => {
+                const { data } = await runQuery(BASIC_QUERY,
+                    { filter: { modified_on: {
+                        gte: '2020-03-10T08:07:00.000000Z'
+                    }}}
+                );
+                data.hosts.data.should.have.length(1);
+                data.hosts.data[0].id.should.equal(TEST_ORG_ID_HOST_IDS[4]);
+            });
+
```

Comment:
I think you would also want to test the strictly great and lesser than in addition to equals for the gte and lte cases as well as some negative equivalents to "assertNotEqual" to be sure it works as expected for all cases, not just one for each operator.

Reply:
@CodeHeeler `modified_on` is using the same `FilterTimestamp` type that's already tested a few times, so I wasn't sure if it was necessary to test it beyond the most common use-cases. I can add a couple more tests if you think it's worth it though

Reply:
What do you mean by negative equivalents in this case, though? Each of them is looking for one specific seeded host. If only one host is returned, that to me seems like it covers the negative as well (all of the other hosts were _not_ returned). wdyt?

Reply:
I'm ok with leaving this as is then.  Thanks for the explanation.

Reply:
Ah, yes, you're right, we're testing the negatives as well by the omission of the other hosts, that's true.

Reply:
I added separate LT & GT tests just to make sure we're covered.

Reply:
Always better to have more test coverage than too little.  Thank you for this.